src/utils/timers.ts

```diff
@@ -57,9 +62,13 @@ function timeEndImpl(label: string, level: number = 3) {
 
 export function getTimings(): SerializedTimings {
 	const newTimings: SerializedTimings = {};
-	Object.keys(timers).forEach(label => {
-		newTimings[label] = timers[label].time;
-	});
+	Object.keys(timers)
+		.sort(
+			(a, b) => getNormalizedStartTime(timers[a].first) - getNormalizedStartTime(timers[b].first)
+		)
+		.forEach(label => {
+			newTimings[label] = timers[label].time;
+		});
```

Comment:
Did this sorting ever change anything for you? As the entries are added to timers as soon as `timeStart` is called for the first time and `Object.keys` implicitly iterates keys in the order they are added, I do not think tracking start times or sorting the entries should be necessary. Or am I overlooking something here which is relevant for watch mode?

Reply:
@lukastaegert whoops, you're right -- sorry, I'd intended to revert this before pushing but forgot.  Fixed now!